SPRNetTool/Domain/BitmapDisplayManager.cs

```diff
@@ -319,6 +320,7 @@ private async Task PlayAnimation()
                                 , SprWorkManager.FileHead.GlobalHeight, PixelFormats.Bgra32))
                             .Also((it) => it.Freeze()));
                     DisplayedBitmapSourceCache.DisplayedBitmapSource = DisplayedBitmapSourceCache.AnimationSourceCaching[frameIndex];
+                    DisplayedBitmapSourceCache.DisplayedColorSource = DisplayedBitmapSourceCache.DisplayedBitmapSource?.Let(it => this.CountColors(it));
```

Comment:
Bỏ đoạn này ra ngoài vòng while thì sao e?

Reply:
Đoạn DisplayedColorSource em chuyển ra ngoài vòng while thì thấy không sao ạ, còn đoạn DisplayedBitmapSource chuyển ra sẽ không thay đổi hình ảnh trên view khi play animation.

Reply:
bỏ ra ngoài vòng while đi e, hoặc e làm luôn cho a 1 cái cache lưu lại color source, chứ mỗi lần lại chạy lại hàm CountColors sẽ bị ảnh hưởng performance
làm theo phương án cache 
có cái TODO ở đó rồi đấy
// TODO : Add array DisplayedColorSource cache for multi frame of spr animation